src/components/AddItems.jsx

```diff
@@ -0,0 +1,102 @@
+import { useCallback, useState } from 'react';
+import { useStateWithStorage } from '../utils';
+import { addItem } from '../api';
+
+const soonDate = 7;
+const kindOfSoonDate = 14;
+const notSoonDate = 30;
+
+export function AddItems() {
+	const [daysUntilNextPurchase, setDaysUntilNextPurchase] = useState(null);
+	const [itemName, setItemName] = useState('');
+
+	const [listPath] = useStateWithStorage('tcl-shopping-list-path', null);
+
+	const handleTextChange = (event) => {
+		return setItemName(event.target.value);
+	};
```

Comment:
Since this function is only returning a single expression, we can simplify it by using an implicit return! 🚀

Reply:
updated!